crates/ruff_server/src/server.rs

```diff
@@ -196,6 +196,7 @@ impl Server {
                     code_action_kinds: Some(vec![
                         CodeActionKind::QUICKFIX,
                         CodeActionKind::SOURCE_ORGANIZE_IMPORTS,
+                        CodeActionKind::SOURCE_FIX_ALL,
```

Comment:
We will need to advertise to the client that we support Ruff scoped code actions during initialization. Refer https://github.com/astral-sh/ruff-lsp/blob/187d7790be0783b9ac41ce025a724cf389bf575c/ruff_lsp/server.py#L852-L869